values.yaml

```diff
@@ -249,9 +254,13 @@ jenkins:
             Args: "cat"
             Tty: true
 docker-registry:
-  persistence:
-    size: 300Gi
+  enabled: false
+
 nexus:
+  service:
+    annotations:
+      fabric8.io/expose: "true"
+      fabric8.io/ingress.annotations: "kubernetes.io/ingress.class: nginx\nnginx.ingress.kubernetes.io/proxy-body-size: 0"
```

Comment:
Do we need that as we're configuring ourself the ingress?

Reply:
yes i can remove 

` fabric8.io/expose: "true" `

but i have to keep 

`fabric8.io/ingress.annotations: "kubernetes.io/ingress.class: nginx\nnginx.ingress.kubernetes.io/proxy-body-size: 0"`

this section disable the maximum allowed size of the client request body, to prevent push error for large image

Reply:
i can remove this line `fabric8.io/expose: "true"` as you said we don't need it

Reply:
If we don't need it, let's remove it, no?

Reply:
I thinks we're talking about 2 separate ingresses.

We definitely want an inline comment here, for the Nexus ingress annotation, to explain why we need `proxy-body-size: 0`.

Reply:
changes applied